php_memcached.c

```diff
@@ -107,6 +112,9 @@ static int php_memc_list_entry(void) {
 #define MEMC_VAL_COMPRESSED          (1<<0)
 #define MEMC_VAL_COMPRESSION_ZLIB    (1<<1)
 #define MEMC_VAL_COMPRESSION_FASTLZ  (1<<2)
+#ifdef HAVE_ZSTD_H
+#define MEMC_VAL_COMPRESSION_ZSTD    (1<<3)
+#endif
```

Comment:
I think this should be defined even if it's not available, later code can return an error if the user requests a compression type that hasn't been compiled in.

Reply:
true, I agree